shadows/framework/src/main/java/org/robolectric/shadows/ShadowApplicationPackageManager.java

```diff
@@ -2238,16 +2238,16 @@ private Context getContext() {
 
   /** Stub that will always throw. */
   @Implementation(minSdk = S)
-  protected PackageManager.Property getProperty(String propertyName, String packageName)
-      throws NameNotFoundException {
+  protected Object /* PackageManager.Property */ getProperty(
+      String propertyName, String packageName) throws NameNotFoundException {
     // TODO: in future read this value from parsed manifest
     throw new NameNotFoundException("unsupported");
   }
 
   /** Stub that will always throw. */
   @Implementation(minSdk = S)
-  protected PackageManager.Property getProperty(String propertyName, ComponentName name)
-      throws NameNotFoundException {
+  protected Object /* PackageManager.Property */ getProperty(
+      String propertyName, ComponentName name) throws NameNotFoundException {
```

Comment:
I remember input fields also need to be converted to `Object` if we want to use it with loose signatures?

Reply:
I'll definitely be sure to double check. Right now just seeing if this fixes the failures in https://github.com/robolectric/robolectric/pull/7927